hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OFSPath.java

```diff
@@ -163,6 +163,14 @@ public String getKeyName() {
     return keyName;
   }
 
+  /**
+   * Returns the last element of the KeyName, which can either be a file name
+   * or the leaf directory/bucket.
```

Comment:
is this description accurate? if path = "/vol/buc2", keyName is (empty) and fileName is (empty). It won't be a bucket name.

Reply:
ohh, thanks @jojochuang, you are right, it can't be a bucket :) I updated the comment accordingly